imagepipeline/src/main/java/com/facebook/imagepipeline/request/ImageRequestBuilder.java

```diff
@@ -256,6 +262,18 @@ public ImageRequestBuilder setPostprocessor(Postprocessor postprocessor) {
     return mPostprocessor;
   }
 
+  /** Sets the HTTP headers.
```

Comment:
Thanks for the PR @rigdern.

We just need to add to this doc (and maybe where you declare `mHeaders` too) to make it clear that support for these added headers is dependent on the `NetworkFetcher` being used.

Ideally we'd add support to the basic `HttpUrlConnectionNetworkFetcher` too but I can live without that as long as this is well enough documented.